libs/exec/execv_windows.go

```diff
@@ -0,0 +1,49 @@
+//go:build windows
+
+package exec
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// Note: Windows does not support an execv syscall that replaces the current process.
+// To emulate this, we create a child process, stream the input and output, and return the exit code.
+// ref: https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/execv-wexecv?view=msvc-170
+func execv(opts ExecvOptions) error {
+	path, err := exec.LookPath(opts.Args[0])
+	if err != nil {
+		return fmt.Errorf("looking up %q failed: %w", opts.Args[0], err)
+	}
+
+	cmd := exec.Command(path, opts.Args[1:]...)
+
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	cmd.Dir = opts.Dir
+	cmd.Env = opts.Env
+
+	// Start the child command.
+	err = cmd.Start()
+	if err != nil {
+		return fmt.Errorf(" %s failed: %w", strings.Join(opts.Args, " "), err)
```

Comment:
nit: for copy-pasteable commands we could add something like this
 https://pkg.go.dev/github.com/alessio/shellescape#QuoteCommand (don't know if that library is the best option, just using is as an example).

Reply:
`bundle run somecommand 'single parameter with spaces'` will be printed as `bundle run somecommand single parameter with spaces`

Reply:
Yeah fair point. I don't think the standard library offers something like this (a perplexity search agrees) and I would not want to rely on a third party library for this. 

I think the error is understandable enough, let's skip this.